src/config/wmodules-aws.c

```diff
@@ -236,6 +238,7 @@ int wm_aws_read(const OS_XML *xml, xml_node **nodes, wmodule *module)
                         }
                     } else if (!strcmp(children[j]->element, XML_ACCESS_KEY)) {
                         if (strlen(children[j]->content) != 0) {
+                            mwarn("Deprecated tag <%s> found at module '%s'. This tag was deprecated in 4.4. Please, use a different authentication method. Check %s for more information.", children[j]->element, WM_AWS_CONTEXT.name, AUTHENTICATION_OPTIONS_URL);
```

Comment:
I think that the message string should be placed in a constant too, something like that  
```c
static const char *DEPRECATED_MESSAGE_TEMPLATE = "Deprecated tag <%s> found at module '%s'. This tag was deprecated in 4.4. Please, use a different authentication method. Check %s for more information."
```
And later be used in the four cases

```c
mwarn(DEPRECATED_MESSAGE_TEMPLATE, children[j]->element, WM_AWS_CONTEXT.name, AUTHENTICATION_OPTIONS_URL);
```

Reply:
Done